### DISCUSSION TITLE
What's the supported stripe api version?

### DISCUSSION THREAD

**Discussion by miaoz2001** [pinax/pinax-stripe#646] [Q&A]
#### Issue Summary

This may be a dumb question but I am wondering what the lasted stripe api version supported is?
According to the doc at ```https://pinax-stripe.readthedocs.io/en/latest/user-guide/settings/```
it says: ```PINAX_STRIPE_API_VERSION``` default to ```2015-10-16```.

Is it that old?

I am using '2017-04-06' with pinax-stripe 4.3.1. It looks ok. But Can I upgrade the stripe version to the latest? I need a feature in stripe that force me upgrade to the latest version, but I am not comfortable with that...

**Comment by chasetb** [comment]
I'm curious if anyone has a good answer for this question. I'm still using the default of `2015-10-16`. It would be nice to know if updating to say `2017-12-14` (current Stripe default) would be alright.
